ui/docker-compose.yml

```
@@ -0,0 +1,39 @@
+---
+services:
+  pmm-server:
+    container_name: pmm-server
+    # Temporary till we have arm builds
+    platform: linux/amd64
+    image: perconalab/pmm-server:3-dev-container
+    ports:
+      - 80:9080
+      - 443:8443
+    volumes:
+      - ./pmm-dev.conf:/etc/nginx/conf.d/pmm-dev.conf:ro
+
+      # Uncomment to use custom (FE) grafana code
+      #- '../../grafana/public:/usr/share/grafana/public'
+      #- '../../grafana/conf/grafana.local-dev.ini:/usr/share/grafana/conf/defaults.ini'
+    environment:
+      - PMM_DEBUG=1
+      - PERCONA_PORTAL_URL=https://portal-dev.percona.com
+      - PERCONA_TEST_PLATFORM_ADDRESS=https://check-dev.percona.com:443
+      - PERCONA_TEST_PLATFORM_PUBLIC_KEY=RWTkF7Snv08FCboTne4djQfN5qbrLfAjb8SY3/wwEP+X5nUrkxCEvUDJ
```

We are changing env variables in v3 - https://github.com/percona/pmm/pull/2857
```suggestion
      - PMM_DEBUG=1
      - PMM_DEV_PERCONA_PLATFORM_ADDRESS=https://check-dev.percona.com:443
      - PMM_DEV_PERCONA_PLATFORM_PUBLIC_KEY=RWTkF7Snv08FCboTne4djQfN5qbrLfAjb8SY3/wwEP+X5nUrkxCEvUDJ
```

---

Thanks, I'll update them.